src/web/client/src/components/Register/Register.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FormEvent } from "react";
+import { shallow } from "enzyme";
+import { render } from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+import Register from "./Register";
+
+describe("Register component", () => {
+  let registerWrapper: any;
+  let onAuthChangeMock: jest.Mock;
+  let historyMock: object;
+  let eventMock: any;
+  let preventDefaultMock: jest.Mock;
+  beforeEach(() => {
+    // Mock Fetch
+    const mockSuccessResponse = "successfully registered";
+    const mockTextPromise: any = Promise.resolve(mockSuccessResponse);
+    const mockFetchPromise: any = Promise.resolve({
```

Comment:
Promise<string> and Promise

Reply:
It gives me this `Generic type 'Promise<T>' requires 1 type argument(s).`